python/ray/rllib/agents/qmix/qmix_policy_graph.py

```diff
@@ -292,8 +301,8 @@ def to_batches(arr):
     @override(PolicyGraph)
     def get_initial_state(self):
         return [
-            self.model.init_hidden().numpy().squeeze()
-            for _ in range(self.n_agents)
+            s.expand([self.n_agents, -1]).numpy()
```

Comment:
The main change here is that we return a list of `[num_agents, h_size_i] for i in n_state_tensors`, rather than `[h_size] for i in n_agents` previously (which was kind of unnatural and didn't generalize well to multiple state tensors).

Relatedly, I'm regretting allowing multiple hidden state tensors, we probably should have just required them to be fused into one element.